python/data/preprocess/others.py

```diff
@@ -138,3 +138,13 @@ def _points_to_voxel(self, points, voxel_size, point_cloud_range, grid_size, vox
                 num_points_per_voxel[voxel_idx] = curr_num_point + 1
 
         return num_voxels
+
+@op_register
+class MaskAndExpand(OpBase):
+    def __init__(self, proj_mask):
+        self.proj_mask = proj_mask
+
+    def __call__(self, samples, **kwargs):
+        if self.proj_mask is not None:
+            samples *= np.load(self.proj_mask)
```

Comment:
According to the config file, the proj_mask file was saved by previous ops. We'd better pass it through parameters instead of using a file

Reply:
ok, newly version wraps it as a new post op.